src/main/java/webserver/Response.java

```diff
@@ -55,7 +55,7 @@ private String createInitialResponseLine(int statusCode) {
     public String toResponseString() {
         String initialLine = createInitialResponseLine(statusCode);
         String allHeaders = convertHeaderMapToString(headers);
-        String responseString = initialLine + CRLF + allHeaders + CRLF + CRLF + body + "\n";
+        String responseString = initialLine + CRLF + allHeaders + CRLF + CRLF + body + CRLF + "\n";
```

Comment:
Is the CRLF or the \n necessary here?  I'm not seeing it in the spec

https://tools.ietf.org/html/rfc2616#page-39

^^ note how the last thing in `response` is `[message-body]`; it doesn't say you have to have any special ending to the body.

If you're setting `content-length` in the response (which is helpful, but not required), I wonder if the `content-length` header is 3 bytes smaller than the content?  This might lead to things like `curl` giving warnings, or possibly to other clients truncating extra whitespace off the end of the response.